src/DependencyInjection/SonataUserExtension.php

```diff
@@ -117,6 +123,17 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('sonata.user.impersonating', $config['impersonating']);
 
         $this->configureGoogleAuthenticator($config, $container);
+
+        $container->setAlias('sonata.user.util.email_canonicalizer', $config['service']['email_canonicalizer']);
+        $container->setAlias('sonata.user.util.username_canonicalizer', $config['service']['username_canonicalizer']);
+        $container->setAlias('sonata.user.util.token_generator', $config['service']['token_generator']);
+        //$container->setAlias('sonata.user.user_manager', new Alias($config['service']['user_manager'], true));
```

Comment:
?

Reply:
This utils was moved from FOSUserBundle (to keep the same functionality like with FOSUSer). In next PRs we can remove it.

Reply:
I mean, why is there a line commented ?

Reply:
Becouse it this alias is create in line 305. For now I do not know what to do with it.